scss/tech.scss

```diff
@@ -49,3 +49,22 @@ $tech-logo-size: 100px;
   object-fit: fill;
   width: $tech-logo-size;
 }
+
+ @media only screen and (max-width: 1024px) {
```

Comment:
this selector and all it's properties are not correctly indented, remember you must use 2 spaces as indentation.